doc/testing.md

```diff
@@ -0,0 +1,7 @@
+# Testing
+
+After launching Hagall, it's a good idea to take a look at the logs to make sure your server is registered, working and accessible. There are a few things you can look for:
+
+* "hagall is successfully registered to hds" should show up in the log
+* `"message":"new client is connected","tags":{"app-key":"0xSMOKE"}}` should show up in the log. These are health checks (also known as smoke tests) running from the central Hagall Discovery Service (HDS) to test that the server is working
+* You can also check metrics like `ws_connected_clients`, see more [Metrics](metrics.md).
```

Comment:
```suggestion
* Check metrics like `ws_connected_clients` (see the [Metrics](metrics.md) document for details)
* Check that the server appears on the posemesh [dashboard](https://dashboard.posemesh.org/servers)
```

Reply:
updated and merged with [Troubleshooting](https://github.com/aukilabs/hagall/pull/7/files/a838c775a4f517d486d49240740626de19f82319#r1495446053)